testsuite/cluster_test/src/instance.rs

```diff
@@ -22,7 +22,12 @@ impl Instance {
         S: AsRef<OsStr>,
     {
         let ssh_dest = format!("ec2-user@{}", self.ip);
-        let ssh_args = vec!["-i", "/libra_rsa", ssh_dest.as_str()];
+        let ssh_args = vec![
+            "-i",
+            "/libra_rsa",
+            "-oStrictHostKeyChecking=no",
```

Comment:
Is it possible that 
`ssh -o UserKnownHostsFile=/foo/bar user@host`
where `bar` depends on the value of the current workspace, would be more secure here?

Reply:
@huitseeker with our current setup only way to get this `/foo/bar` file is to go on each validator, ask them what is their key and trust (for the first time at least) that they told the truth. This is somewhat more secure than trusting *every* time, but since we already have test environment isolated into private VPC, I think this is not necessary.

Reply:
I was more thinking of setting the file in question in a user-writeable directory on the first connection, and then reusing this file until the next workplace deletion.